lib/activejob/lockable/lockable.rb

```diff
@@ -58,7 +65,11 @@ def locked_ttl
       private
 
       def lock_period
-        options[:lock]
+        options&.fetch(:lock, 0).to_i
+      end
+
+      def lock_extra_info
+        "[key #{lock_key}] [seconds #{lock_period.to_i}] [job_id #{self.job_id}] [class_name: #{self.class}]"
```

Comment:
I would like us to change the way we are logging from this, given we use it in our Rails applications.

We convert log messages to JSON, and it would be beneficial to search and filter by these values directly. So I would suggest returning a hash of:

```ruby
{
  message: lock_extra_info,
  active_lock: { 
    lock_key: lock_key,
    lock_period: lock_period,
    job_id: job_id,
    class_name: self.class,
}
```

wdyt?
```

Reply:
I think this log would appear in the `event.message` field, and what we expect is a string message.

Actually I'm ok with that.

Reply:
Can you open an issue about this? There are a few things we might want to consider, like "does it work?" and "this is a public library, should we enforce that format?"

Reply:
Sure, since this is public we can leave it just returning the simple message for now :)